index.js

```diff
@@ -50,209 +54,218 @@
 import {visit} from 'unist-util-visit'
 import {toString} from 'nlcst-to-string'
 
-const closingQuotes = {'"': '”', "'": '’'}
-const openingQuotes = {'"': '“', "'": '‘'}
-
-const educators = {
-  dashes: {
-    /**
-     * Transform two dahes into an em-dash.
-     *
-     * @type {Method}
-     */
-    true(node) {
-      if (node.value === '--') {
-        node.value = '—'
-      }
-    },
-    /**
-     * Transform three dahes into an em-dash, and two into an en-dash.
-     *
-     * @type {Method}
-     */
-    oldschool(node) {
-      if (node.value === '---') {
-        node.value = '—'
-      } else if (node.value === '--') {
-        node.value = '–'
+const defaultClosingQuotes = {'"': '”', "'": '’'}
+const defaultOpeningQuotes = {'"': '“', "'": '‘'}
+
+function createEducators(/** @type {Options} */ options = {}) {
```

Comment:
```suggestion
/**
 * @param {Options} options
 */
function createEducators(options) {
```

We also probably do `options || {}` somewhere in the plugin already, if that value passed is this `createEducators`’s options don’t need a default